### PR TITLE
abdalla/add markets section

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by mustofa-binary — APPROVED** [review]

File: src/components/elements/carousel/carousel-style.js [hunk 1]
```diff
@@ -5,13 +5,20 @@ import { device } from 'themes'
...
     outline: 0;
     cursor: pointer;
 `
 
+export const ReviewsButton = styled(StyledDots)`
+    background-color: ${props => (props.selected ? '#999999' : '#e5e5e5')};
+`
+
+export const MarketsButton = styled(StyledDots)`
+    background-color: ${props => (props.selected ? '#ffc71b' : '#e5e5e5')};
```
Comment (mustofa-binary) [inline src/components/elements/carousel/carousel-style.js]: please use theme, if color doesnt have, pls add to the theme

File: src/components/elements/carousel/carousel.js [hunk 2]
```diff
@@ -10,7 +10,6 @@ import {
     Container,
     Slide,
 } from './carousel-style'
-//import { Image } from 'components/elements'
```
Comment (mustofa-binary) [inline src/components/elements/carousel/carousel.js:13]: can remove this comment

File: src/pages/home/_trade-markets.js [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+                <CardHeader>{title}</CardHeader>
+            </Flex>
+            <CardContent>{content}</CardContent>
+        </CardContainer>
+    )
+}
+
+export const TradeMarkets = () => {
+    const forex = {
+        title: 'Forex',
```
Comment (mustofa-binary) [inline src/pages/home/_trade-markets.js]: <Localize

**Review by mustofa-binary — CHANGES_REQUESTED** [review]

File: src/pages/home/_trade-markets.js (hunk 3 above)
Comment (ghost) [reply src/pages/home/_trade-markets.js]: Do I need to localize the title as well or only the content?

**Review by mustofa-binary — APPROVED** [review]

**Pull request merged**